core/sds-aspect-meta-model-python/tests/resources/constraints/AspectWithConstrainedCollection.ttl

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2021 Robert Bosch Manufacturing Solutions GmbH
```

Comment:
The year in the copy right headers in the code files is 2022. Given that this is the initial contribution, IMHO we should have the same year in all files.